lib/LinkEmbedder/Link.pm

```diff
@@ -382,9 +383,13 @@ __DATA__
 <a class="le-<%= $l->type %> le-provider-<%= lc $l->provider_name %>" href="<%= $l->url %>"><%= Mojo::Util::url_unescape($l->url) %></a>
 % }
 @@ video.html.ep
-<video class="le-<%= $l->type %> le-provider-<%= lc $l->provider_name %>" height="640" width="480" preload="metadata" controls>
-% for my $s (@{$l->{sources} || []}) {
-  <source src="<%= $s->{url} %>" type="<%= $s->{type} || '' %>">
-% }
-  <p>Your browser does not support the video tag.</p>
-</video>
+@@ video.html.ep
+<div class="le-<%= $l->type %> le-provider-<%= lc $l->provider_name %>" height="640" width="480" preload="metadata" controls>
+  <video class="le-<%= $l->type %> le-provider-<%= lc $l->provider_name %>" height="640" width="480" preload="metadata" controls>
```

Comment:
It will be a breaking change, but I think we should remove the classes here and just keep them on the div.

Reply:
Makes sense.
I'm wondering if we keep the height/width in the video tag or not...

Reply:
I removed the width/height from the video tag, and adding max-width and width to .le-video allows to control the size.